packages/premint-sdk/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@zoralabs/premint-sdk",
+  "version": "0.0.2-premint-api.2",
+  "repository": "https://github.com/ourzora/zora-protocol",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "scripts": {
+    "build": "tsup",
+    "prepack": "yarn build",
+    "test": "vitest src",
+    "generate-types": "npx openapi-typescript https://api.zora.co/premint/openapi.json -o src/generated/premint-api-types.ts && npx openapi-typescript https://api.zora.co/discover/openapi.json -o src/generated/discover-api-types.ts",
+    "anvil": "source .env.anvil && anvil --fork-url $FORK_RPC_URL --fork-block-number $FORK_BLOCK_NUMBER --chain-id 31337"
+  },
+  "dependencies": {
+    "@zoralabs/zora-1155-contracts": "2.1.1-premint-api.0",
```

Comment:
this should be:
`"@zoralabs/zora-1155-contracts": "*"`

Reply:
Changesets changes this on release.